apps/ccd/ccd-data-store-api/aat.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         CCD_DOCUMENT_URL_PATTERN: ^https?://(((?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|(dm-store-aat\.service\.core-compute-aat\.internal|dm-store-(pr-[0-9]+|preview)(\.service\.core-compute-preview\.internal|\.preview\.platform\.hmcts\.net))(?::\d+)?)\/documents\/[A-Za-z0-9-]+(?:\/binary)?)|((em-hrs-api-(?:aat|staging)\.service\.core-compute-aat\.internal|em-hrs-api-staging\.aat\.platform\.hmcts\.net|em-hrs-api-(pr-[0-9]+|preview)(\.service\.core-compute-preview\.internal|\.preview\.platform\.hmcts\.net))(?::\d+)?\/hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9]+))
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,divorce_frontend,civil_service,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,probate_backend,et_cos,civil_general_applications
         DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-aat.postgres.database.azure.com
-        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,wa_task_configuration_api,divorce_frontend,wa_task_management_api,wa_workflow_api,em_hrs_api,nfdiv_case_api,civil_service,wa_task_monitor,ccd_case_document_am_api,adoption_cos_api,adoption_web,et_cos,et_msg_handler,prl_cos_api,wa_case_event_handler,et_sya_api,ds_ui,prl_citizen_frontend,hmc_cft_hearing_service,prl_cos_api,fis_hmc_api,ccd_next_hearing_date_updater,civil_general_applications,fis_cos_api,sptribs_case_api
+        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,wa_task_configuration_api,divorce_frontend,wa_task_management_api,wa_workflow_api,em_hrs_api,nfdiv_case_api,civil_service,wa_task_monitor,ccd_case_document_am_api,adoption_cos_api,adoption_web,et_cos,et_msg_handler,prl_cos_api,wa_case_event_handler,et_sya_api,ds_ui,prl_citizen_frontend,hmc_cft_hearing_service,prl_cos_api,fis_hmc_api,ccd_next_hearing_date_updater,civil_general_applications,fis_cos_api,sptribs_case_api,et-wa-post-deployment-ft-tests
```

Comment:
As per standard, please use underscore instead of hyphen in the service name.

Reply:
Thanks Rajaram. I have updated this.

Reply:
Still I can see the service name (et-wa-post-deployment-ft-test) with hyphen

Reply:
Sorry, got that change pushed now